R/data.R

```diff
@@ -1,54 +1,54 @@
 #' Geological Time Scale 2020
 #'
-#' A dataset of the Geological Time Scale 2020. Age data from:
-#'  \url{https://stratigraphy.org/timescale/}.
-#' Supplementary information is also included in the dataset for plotting functionality (e.g. GTS2020 colour scheme).
+#' A dataframe of the Geological Timescale 2020. Age data from:
+#'  \url{https://stratigraphy.org/ICSchart/ChronostratChart2020-03.pdf}.
+#' Supplementary information is included in the dataset for plotting functionality (e.g. GTS2020 colour scheme).
 #'
 #' @format A data frame with 189 rows and 12 variables:
 #' \describe{
-#'   \item{index}{Index number for the temporal order of all intervals present in the dataset}
-#'   \item{stage_number}{Index number for stages}
-#'   \item{series_number}{Index number for series}
-#'   \item{system_number}{Index number for system}
-#'   \item{interval_name}{Names of intervals in the dataset}
-#'   \item{rank}{The temporal rank of intervals in the dataset}
-#'   \item{max_ma}{The maximum age of the interval in millions of years before present}
-#'   \item{mid_ma}{The midpoint age of the interval in millions of years before present}
-#'   \item{min_ma}{The minimum age of the interval in millions of years before present}
-#'   \item{duration_myr}{The duration of the interval in millions of years}
+#'   \item{index}{Index number for the temporal order of all intervals present in the dataset.}
+#'   \item{stage_number}{Index number for stages.}
+#'   \item{series_number}{Index number for series.}
+#'   \item{system_number}{Index number for systems.}
```

Comment:
I don't completely disagree, but these columns do make it very easy for the user to sort the data (one step vs two). The values also make sense once you've subset to whatever desired rank, rather than having numbers such as: 5, 11, 17, etc. With these columns you can also sort the data without the need to subset. I think this will be useful for people who might just extract the dataframe from the package (to excel) or visualise in R (some users will want to work way more interactively than others). I don't think this is necessarily doing any harm at this time, but perhaps something we want to consider dropping if package size becomes an issue (these files are 4kb each currently).

Reply:
Maybe there was a miscommunication, the "interval number" column would just be the three existing columns combined (so the number would reset to 1 for each rank). This way the number should just be 1 to n when you subset to a specific rank. Also, it's not clear to me why you would ever WANT to sort the data without subsetting first; what use is having the other ranks still in the data?